doc/content/examples/example10a.md

```diff
@@ -0,0 +1,63 @@
+# Example 10a: Cantilever example demonstrating central difference time integration
+
+## Model Description
+
+This example demonstrates the central difference time integration in MASTODON. A 2 x 2 x 20 $ft$ cantilever is modeled with the bottom surface fixed along x, y & z directions. A uniform normal pressure of 1 $kip/ft^2$ is applied to the top surface along the axis of the model. The resulting stress time-history and displacement time-history are then obtained at locations (0, 0, 10) and (0, 0, 20) respectively and compared with results from Implicit Newmark-Beta time integration.
+
+[fig:ex10a_input] demonstrates the overall dimensions and boundary conditions of the model.
+
+!media media/examples/MASTODON_ex10_model.png
+       style=width:50%;margin-left:150px;float:center;
+       id=fig:ex10a_input
+       caption=Input model in MASTODON
+
+## Modeling in MASTODON
+
+!listing examples/ex10a/MASTODON_Explicit_HEX8.i
+
+The automeshing option provided by MASTODON is used to create the geometry and specify the element type (see documentation of [GeneratedMesh](syntax/index.md) in Syntax for detailed information).
```

Comment:
Please change the syntax page link. You can link to mesh/GeneratedMesh.md instead.